packages/gatsby-core-utils/src/ci.ts

```diff
@@ -1,13 +1,16 @@
 import ci from "ci-info"
 
 const CI_DEFINITIONS = [
-  envFromCIAndCIName,
   getEnvDetect({ key: `NOW_BUILDER_ANNOTATE`, name: `ZEIT Now` }),
   getEnvDetect({ key: `NOW_REGION`, name: `ZEIT Now v1` }),
   getEnvDetect({ key: `VERCEL_URL`, name: `Vercel Now` }),
   getEnvDetect({ key: `NOW_BUILDER`, name: `Vercel Now` }),
   getEnvDetect({ key: `VERCEL_BUILDER`, name: `Vercel Now` }),
   getEnvDetect({ key: `CODESANDBOX_SSE`, name: `CodeSandbox` }),
+  getEnvDetect({ key: `GITHUB_ACTIONS`, name: `GitHub Actions` }),
```

Comment:
Documented in https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables (at least as of moment of writing this comment)